java/ql/test/library-tests/dataflow/external-models/steps.expected

```diff
@@ -1,4 +1,5 @@
 invalidModelRow
+| Dubious name "getPlatformTypeUpperBound$kotlin_stdlib" in summary model. |
```

Comment:
This is an `internal` property. Its modifiers are `final`, `internal`, and `public`. The latter one is causing this to show up in the MaD generation. I'll have to investigate where the `public` is coming from. Is it possible that we extract the modifiers in both kotlin and java, and `internal` is actually `public` when viewed from java?

Reply:
Have you had a chance to look into this? In any case, it might be a good idea to exclude this from model generation of there is any doubt.

Reply:
We extract `getPlatformTypeUpperBound$kotlin_stdlib` as internal in kotlin and public in java. So effectively it's public and therefore shows up in the model generation. I asked internally the team how to move on with this.

Reply:
This will be fixed by https://github.com/github/codeql/pull/10221